aten/src/ATen/native/vulkan/Vulkan.cpp

```diff
@@ -989,7 +989,8 @@ void copy_buffer_to_image(const VBuffer& buffer, VImage& image) {
     int32_t w;
     int32_t h;
   };
-  ConstBlock constBlock{image.w(), image.h()};
+  ConstBlock constBlock{static_cast<int32_t>(image.w()),
+                        static_cast<int32_t>(image.h())};
```

Comment:
I think safe_downcast is a better choice here but that might requires some refactoring to move it out of Pool.h.